src/mscorlib/shared/System/Globalization/CompareInfo.cs

```diff
@@ -389,38 +389,20 @@ internal int Compare(ReadOnlySpan<char> string1, string string2, CompareOptions
             return CompareString(string1, string2, options);
         }
 
-        // TODO https://github.com/dotnet/corefx/issues/21395: Expose this publicly?
-        internal virtual int Compare(ReadOnlySpan<char> string1, ReadOnlySpan<char> string2, CompareOptions options)
+        internal virtual int CompareOptionNone(ReadOnlySpan<char> string1, ReadOnlySpan<char> string2, CompareOptions options)
```

Comment:
Why take take an options and force it to be None if it's not going to be used?

Reply:
We need to pass the option to CompareString which passes it to the interop call.

Reply:
Hmm... I just realized what you meant. Lol.

We can just do the following:
```C#
        internal virtual int CompareOptionNone(ReadOnlySpan<char> string1, ReadOnlySpan<char> string2)
        {
            return _invariantMode ?
                string.CompareOrdinal(string1, string2) :
                CompareString(string1, string2, CompareOptions.None);
        }
```